Sources/libcggen/BCCGGenerator.swift

```diff
@@ -0,0 +1,341 @@
+import BCCommon
+import Foundation
+
+private protocol ByteCodable {
+  var byteCode: [UInt8] { get }
+}
+
+extension UInt32: ByteCodable {
+  var byteCode: [UInt8] {
+    var littleEndian = self.littleEndian
+    let count = MemoryLayout<Self>.size
+    let bytePtr = withUnsafePointer(to: &littleEndian) { pointer in
+      pointer.withMemoryRebound(to: UInt8.self, capacity: count) { pt in
+        UnsafeBufferPointer(start: pt, count: count)
+      }
+    }
+    return Array(bytePtr)
+  }
+}
+
+extension Int: ByteCodable {
+  var byteCode: [UInt8] {
+    UInt32(self).byteCode
```

Comment:
That a dangerous thing to do. What if `Int` would be negative?

Reply:
Can we eliminate this function and keep explicit conversion where it's actually needed?